pages/linux/resolvconf.md

```diff
@@ -0,0 +1,29 @@
+# resolvconf
+
+> Manage nameserver information.
+> Acts as an intermediary between rograms that supply nameserver information and applications that use this information.
+> More information: <https://manpages.ubuntu.com/manpages/resolvconf.8.html>.
+
+- Add or override a record (IFACE.PROG) and run the update scripts if updating is enabled:
```

Comment:
Are you sure that the record is written as `IFACE.PROG` (or `interface<dot>program` as far as I can understood)? Some manpages refer this as `interface<dot>protocol`, like `dhcp` and `ppp`.

Reply:
Oh I see that this is caused by the difference between Debian or openresolv.